### PR TITLE
Address memory issue caused by open api generator

### PR DESCRIPTION
**Pull request by jackjii79**
https://h2oai.slack.com/archives/G01C9KKQLAC/p1710175541471659

The root cause of performance issue is due to auto generated api model class should using `Row` class instead of `ArrayList<String>`, which makes JVM spend more than necessary heap size to do the class conversion results in slowness and abnormal memory increasing. 
This PR also removes deprecated sub gradle project from building pipeline

### CODE REVIEW

**Review by g-eoj — COMMENTED** [review]
Thanks! I don't see any issues but prefer someone with more Java experience to give approval.

File: settings.gradle [hunk 1]
```diff
@@ -14,18 +14,18 @@ dependencyResolutionManagement {
 }
 
 rootProject.name = 'dai-deployment-templates'
-include 'aws-lambda-scorer:lambda-template'
-include 'aws-lambda-scorer:terraform-recipe'
-include 'common:jdbc'
+//include 'aws-lambda-scorer:lambda-template'
```
Comment (jackjii79) [inline settings.gradle:17]: All deprecated scorer which not only takes long time to build also no longer used by MLOps

**Comment by jackjii79** [comment]
E2E: http://jenkins:8080/view/Model%20Ops/job/mlops/job/environments-helm/job/jh-helm/job/manual/job/test-mojo/allure/

**Review by shsma — APPROVED** [review]
LGTM! Thanks @jackjii79

**Pull request merged**